### DISCUSSION TITLE
TTS with voice cloning in Kannada

### DISCUSSION THREAD

**Discussion by Ashish301101** [coqui-ai/TTS#2049] [General Q&A] [answered]
Hi everyone. I am Ashish H and I am working on a project which takes the input as english video and then generates the audio in a target language kannada and at the same time maintains the voice of the english speaker in the video. Can i get some help about how can i go about this process and how to train the model and which dataset to use for it.

**Comment by Ca-ressemble-a-du-fake** [comment, marked as answer]
Hi Ashish,

What an exciting project! I give you some hints / broad answers. To train the model did you read the [doc](https://tts.readthedocs.io/en/latest/training_a_model.html). If there is no datasets for Kannada you may have to create your own one. 

I don't know if it could work because of the different character set that Kannada has, but you could fine tune their English model (YourTTS multilingual released model) which would give you better results faster.
